modules/nw-modifying-operator-install-config.adoc

```diff
@@ -81,33 +90,35 @@ cluster-network-02-config.yml
 cluster-network-03-config.yml
 ----
 
-. Open the `cluster-network-03-config.yml` file in an editor and enter a custom resource (CR) that
-describes the Operator configuration you want:
+. Open the `cluster-network-03-config.yml` file in an editor and specify the advanced network configuration for your cluster, such as in the following examples:
 +
+--
+.Specify a different VXLAN port for the OpenShift SDN network provider
 [source,yaml]
 ----
 apiVersion: operator.openshift.io/v1
 kind: Network
 metadata:
   name: cluster
-spec: <1>
-  clusterNetwork:
-  - cidr: 10.128.0.0/14
-    hostPrefix: 23
-  serviceNetwork:
-  - 172.30.0.0/16
```

Comment:
yes, this is good. The user should not be re-specifying the parts that are already in install-config

Reply:
@danwinship, cool, I've been wanting to rewrite this to eliminate that confusion for awhile now.